documents/operating.md

```diff
@@ -29,15 +29,113 @@ SECTION 2: 🤝🤝🤝 Partnership 🤝🤝🤝
 ---------------------------------------------------------------------
 SECTION 3: 🤔🤔🤔 Projects 🤔🤔🤔
 ---------------------------------------------------------------------
-- [ ] Types of project
-- [ ] Evaluating ideas
-- [ ] IP (Intellectual Property / Internet Protocol)
-- [ ] Open source
-- [ ] When a project should be considered retired
-- [ ] Removing a project from BIFFUD
+- [x] Types of project
+- [x] Evaluating ideas
+- [x] IP (Intellectual Property / Internet Protocol)
+- [x] Open source
+- [x] When a project should be considered retired
+- [x] Removing a project from BIFFUD
 - [ ] Adopting IP from existing entities / projects
 - [ ] Reallocating copyright / IP upon spinoff
 
+### BIFFUD Projects
+BIFFUD exists to support the creation of projects.  These projects are
+approved by the Corporate Overlords, owned and maintained by Members,
+and built by Minions.
+
+### Ownership Contracts
+All owners of a project must be BIFFUD members, and the percentages and specific
+terms of ownership must be coordinated by those members as part of the project's
+induction into The Company in the form of an Ownership Contract.
+
+The Ownership Contract should specify participation expectations
+and ownership percentages, as well as guidelines for how potential profits
+should be distributed.
+
+Changes to project ownership over time must be communicated to The Beastmaster.
```

Comment:
What kind of requirement do you have in mind?

The idea I was running with when writing this section is that the fewer requirements for projects the better and that the important thing from BIFFUD corporate when it comes to project ownership is how to distribute potential profits at the end of the year.  In a sense, it's up to the project owners to decide with how they want ownership terms to work for their specific project.

That said I think we will wanna put together some example contract(s?) that we recommend as having been thought through.